docsets/Discord4J/docset.json

```diff
@@ -1,14 +1,42 @@
 {
     "name": "Discord4J",
-    "version": "2.7.0",
+    "version": "2.9.1",
     "archive": "Discord4J.tgz",
```

Comment:
You need to update the main archive as well.

Reply:
Fixed in #3eb2f7a